### PR TITLE
Import api and filter api tests

### PR DESCRIPTION
**Pull request by pathimanoj**
### What

Added Import api and filter api tests.
There are 2 tests failing in import api and bugs are raised for those.
There are 3 tests failing in import api and bugs are raised for those.

### How to review

Run the tests and review the code.

### Who can review

Anyone.

### CODE REVIEW

File: config/config.go [hunk 1]
```diff
@@ -0,0 +1,9 @@
+package config
+
+func ImportAPIURL() string {
+	return "http://localhost:21800"
+}
+
+func FilterAPIURL() string {
+	return "http://localhost:22100"
```
Comment (CarlHembrough) [inline config/config.go:8]: These values will need to be configurable - see the pattern used in other cmd repos: https://github.com/ONSdigital/dp-observation-importer/blob/cmd-develop/config/config.go

**Comment by CarlHembrough** [comment]
Consider taking out the httpexpect directory, and just have the service name directories at the root of the repo.

**Comment by CarlHembrough** [comment]
govendor needs running to pull in addional dependencies. See https://github.com/kardianos/govendor and ask any questions 👍

**Review by CarlHembrough — CHANGES_REQUESTED** [review]
Looks good overall. I have added a few comments. I would suggest using a consistent 'given, when, then' structure for documenting tests to improve their readability. This will also improve the reporting.

**Comment by nshumoogum** [comment]
This PR has been superceded by other branches and pull requests

**Comment by nshumoogum** [comment]
This PR has been superceded by other branches and pull requests

**Pull request closed without merging**